### ISSUE TITLE
Question: Relationship with `desktop-save` and `desktop-read`

### ISSUE THREAD

**Issue by rajivr** [alphapapa/burly.el#11]
Hi @alphapapa 

I currently use `desktop-save` and `desktop-read` in order to save and restore frames when I have to restart emacs.

I was wondering if `blury.el` does something similar?

I've just started exploring `org-ql`. Thanks for that.

**Comment by alphapapa** [comment]
Hi,

> I was wondering if blury.el does something similar?

I don't know what `blury.el` does.  ;)  As for `burly`, I've tried to explain what it does in the readme.  Please let me know if it seems to be missing information or is hard to follow, or if you have more specific questions that aren't addressed in it.

> I've just started exploring org-ql. Thanks for that.

Thanks.  Please let me know if you have any feedback.

**Comment by rajivr** [comment]
Hi,

> I don't know what `blury.el` does. ;) As for `burly`, I've tried to explain what it does in the readme. Please let me know if it seems to be missing information or is hard to follow, or if you have more specific questions that aren't addressed in it.

Thanks for the kind nudge to checkout the readme. I missed the sentence - "In terms of built-in features, Burly may be seen as integrating the built-in libraries bookmark.el and frameset.el, while also replacing and improving on some of the functionality of desktop.el."

I think if this sentence had come after in the first paragraph, before talking about the internals, that would have been slightly better for me.

I'll give definitely give `burly.el` a try very soon, and will also use the opportunity to understand Emacs' `bookmark.el` and `frameset.el` libraries. 

Thanks again!

**Comment by alphapapa** [comment]
> I think if this sentence had come after in the first paragraph, before talking about the internals, that would have been slightly better for me.

Yeah, I originally had it in the first paragraph, but I moved it to the end of the intro.  I want Burly to be interpreted primary on its own terms.  And I don't want to give the impression that Burly actually *uses* `desktop.el` or actually *replaces* parts of it.  I should probably rewrite that sentence.

**Comment by alphapapa** [comment]
I decided to remove mention of `desktop.el` from the docs.  I think that was more likely to confuse than help users.
